rust/transaction_signing/src/tests.rs

```diff
@@ -103,11 +126,43 @@ fn print_db_content(dbname: &str) -> String {
         let verifier_key = VerifierKey::from_ivec(&verifier_key_vec);
```

Comment:
Verifier details should be eventually simplified here to `general` and `custom(..)`. Identicons are now tested in `navigator`, json-like syntax is not used anywhere else.